source/lac/trilinos_vector.cc

```diff
@@ -562,24 +578,11 @@ namespace TrilinosWrappers
 
   void
   Vector::reinit (const size_type n,
-                  const bool      omit_zeroing_entries)
+                  const bool      )
   {
-    if (size() != n)
-      {
-        Epetra_LocalMap map ((TrilinosWrappers::types::int_type)n, 0,
-                             Utilities::Trilinos::comm_self());
-        vector.reset (new Epetra_FEVector (map));
-      }
-    else if (omit_zeroing_entries == false)
-      {
-        int ierr;
-        ierr = vector->GlobalAssemble(last_action);
-        (void)ierr;
-        Assert (ierr == 0, ExcTrilinosError(ierr));
-
-        ierr = vector->PutScalar(0.0);
-        Assert (ierr == 0, ExcTrilinosError(ierr));
-      }
+    Epetra_LocalMap map ((TrilinosWrappers::types::int_type)n, 0,
+                         Utilities::Trilinos::comm_self());
+    vector.reset (new Epetra_FEVector (map));
```

Comment:
There was probably no danger here in this function, given that we're working in the non-MPI case. But the cost is likely negligible.